frontend/src/tests/components/eMIB/Email.test.js

```diff
@@ -14,22 +15,40 @@ const emailStub = {
 const hasAction = <i className="fas fa-sign-out-alt" style={{ color: "#00565E" }} />;
 
 it("default email renders with subject as an h3", () => {
-  const wrapper = shallow(<Email email={emailStub} emailCount={0} taskCount={0} />);
+  const wrapper = shallow(
+    <Email email={emailStub} emailCount={0} taskCount={0} emailActions={[]} />
+  );
   const subject = <h3>Subject 1</h3>;
   expect(wrapper.contains(subject)).toEqual(true);
   expect(wrapper.contains(hasAction)).toEqual(false);
 });
 
 it("shows action when email count is non zero", () => {
-  const wrapper = shallow(<Email email={emailStub} emailCount={1} taskCount={0} />);
+  const wrapper = shallow(
+    <Email email={emailStub} emailCount={1} taskCount={0} emailActions={[]} />
+  );
   const subject = <h3>Subject 1</h3>;
   expect(wrapper.contains(subject)).toEqual(true);
   expect(wrapper.contains(hasAction)).toEqual(true);
 });
 
 it("shows action when task count is non zero", () => {
-  const wrapper = shallow(<Email email={emailStub} emailCount={0} taskCount={2} />);
+  const wrapper = shallow(
+    <Email email={emailStub} emailCount={0} taskCount={2} emailActions={[]} />
+  );
   const subject = <h3>Subject 1</h3>;
   expect(wrapper.contains(subject)).toEqual(true);
   expect(wrapper.contains(hasAction)).toEqual(true);
 });
+
+it("shows as many CollapsingItemContainer as there are actions", () => {
```

Comment:
Is it possible to add a describe block and add another test that proves that if there are two actions, you'll get a second _CollapsingItemContainer_?

Because now, it tests only if you have one.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/181/commits/c39bc7f5681832e92e9dd90f5b069b3378b3d54a